tests/testthat/test-dbi.R

```diff
@@ -36,6 +36,15 @@ test_that("dbGetQuery works with parameterized queries", {
   expect_equal(nrow(virginica), 50)
 })
 
+test_that("dbGetQuery works with native parameterized queries", {
+  if (spark_version(sc) < "3.4.0") skip("native parameterized queries not supported before 3.4.0")
```

Comment:
Can we use `test_requires_version("3.4.0")` instead? I can commit this change if you're ok with that.